components/cosmetic_filters/resources/data/content_cosmetic.ts

```diff
@@ -76,11 +93,14 @@ const idleize = (onIdle: Function, timeout: number) => {
 }
 
 const isRelativeUrl = (url: string): boolean => {
-  return (
-    !url.startsWith('//') &&
-    !url.startsWith('http://') &&
-    !url.startsWith('https://')
-  )
+  try {
+    new URL(url) // tslint:disable-line
```

Comment:
nit: `new URL` will work for all schemes including internal schemes like `brave`. As I understand these scriptlets will only work for `http` `https` URLs, adding a filter for scheme to exclude the internal schemes makes sense.

Reply:
this would also throw exceptions for URLs like: `abc.com/path` which are not a relative URL.

Reply:
let's create a follow up on that, as it's out of the scope of the issue to bring cosmetic filters to Android. I just don't want to blow that PR more

Reply:
Created a follow up https://github.com/brave/brave-browser/issues/13471